nuget/Microsoft.Windows.CsWinRT.targets

```diff
@@ -147,6 +147,44 @@ $(CsWinRTFilters)
     <Copy SourceFiles="$(CsWinRTGeneratedFilesDir)\$(AssemblyName).winmd" DestinationFolder="$(TargetDir)" UseHardlinksIfPossible="false" SkipUnchangedFiles="true" />
   </Target>
 
+  <!-- ====================== -->
+  <!-- Call The GuidPatch tool on the projection .dll -->
```

Comment:
We should give some thought to the name.  Do we want to call it GuidPatch or something like GuidOptimizer or GuidLookupOptimizer?

Reply:
Maybe GuidLookupOptimization?

Reply:
Went with IIDOptimizer